PendingReleaseNotes.md

```diff
@@ -2,15 +2,20 @@
 
 ## Breaking Changes
 
-- The removal of `CSI_ENABLE_READ_AFFINITY` option and its replacement with per-cluster
-read affinity setting in cephCluster CR (CSIDriverOptions section) in [PR](https://github.com/rook/rook/pull/13665)
+- The minimum supported version of Kubernetes is v1.25.
+  Upgrade to Kubernetes v1.25 or higher before upgrading Rook.
+- The Rook operator config `CSI_ENABLE_READ_AFFINITY` was removed. v1.13 clusters that have modified
+  this value to be `"true"` must set the option as desired in each CephCluster as documented
+  [here](https://rook.github.io/docs/rook/v1.14/CRDs/Cluster/ceph-cluster-crd/#csi-driver-options)
```

Comment:
Can we use the relative path? The docs publishing takes care of the rest.
```suggestion
  [here](../CRDs/Cluster/ceph-cluster-crd.md#csi-driver-options)
```

Reply:
Does it matter/work for the pending release notes doc?

Reply:
Oh right, nevermind, ignore my comment since it's not under `Documentation/`